server/server/settings.py

```diff
@@ -52,6 +52,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    'drf_yasg',
```

Comment:
make sure to add the new dependency `drf-yasg`  to requirements.txt

Reply:
👍 thanks for the reminder